### DISCUSSION TITLE
How to change the PF and IPF from scatter to cloud or contour?

### DISCUSSION THREAD

**Discussion by JinkuiMeng** [mtex-toolbox/mtex#801] [Ask Anything]
PF or IPF figures can be drawn with the following code, but they are in the form of scattered points.

....
_%% Plot the pole figure
% the fixed crystal directions_
h = Miller({1,0,0},{0,1,0},{0,0,1},ebsd('Titanium').CS);

_% plot their positions with respect to specimen coordinates_
figure(3)
plotPDF(ebsd('Titanium').orientations,h,'figSize','medium')

_%% plot the IPF figure
% the fixed specimen direction_
r = vector3d.Z;

_% plot the position of the z-Axis in crystal coordinates_
figure(4)
plotIPDF(ebsd('Titanium').orientations,r,'MarkerSize',5,...
  'MarkerFaceAlpha',0.05,'MarkerEdgeAlpha',0.05,'antipodal')
....

How can I get the PF map or IPF in the form of cloud map or contour? such as,
from:
![image](https://user-images.githubusercontent.com/64880536/108712594-f8afbe80-7551-11eb-8448-e39fc63f08a1.png)

To:
![image](https://user-images.githubusercontent.com/64880536/108712312-8dfe8300-7551-11eb-978f-81d180c2d97f.png)
![image](https://user-images.githubusercontent.com/64880536/108712579-f0578380-7551-11eb-9552-e50f5e233cb4.png)

**Comment by zmichels** [comment]
Hi.

You can add the arguments ‘contour’, contourf’, or ‘smooth’... and you can specify halfwidths for those as well.

Try something like this (from your code just added some):

plotPDF(ebsd('Titanium').orientations,h,’smooth’,’halfwidth’,10*degree,'figSize','medium')

**Reply by JinkuiMeng** [reply]
It works, thank you very much!
